### ISSUE TITLE
Invalid aria attribute on profile page

### ISSUE THREAD

**Issue by ashubawork** [ultimatemember/ultimatemember#1456]
Accessibility check of profile page fails due to invalid aria attribute in profile photo HTML.

`<div class="um-profile-photo" data-user_id="123"  aria-invalid=&quot;false&quot; >`
The reason is in includes/core/um-actions-profile.php line 1020:

`<div class="um-profile-photo" data-user_id="<?php echo esc_attr( um_profile_id() ); ?>" <?php echo esc_html( UM()->fields()->aria_valid_attributes( UM()->fields()->is_error( 'profile_photo' ), 'profile_photo' ) ); ?>>`

Using esc_html() for whole attribute HTML instead of esc_attr() only for attribute value causes this problem.

https://wordpress.org/support/topic/bug-invalid-aria-attribute-on-profile-page/#post-17401604
